### DISCUSSION TITLE
BigQuad Deck firmware

### DISCUSSION THREAD

**Discussion by cc299792458** [bitcraze/discussions#269] [Q&A] [answered]
Hi, Bitcraze!
I want to use BigQuad Deck with my crazyflie. I follow usage in it's website https://www.bitcraze.io/products/bigquad-deck/.
In section Building Firmware-Enable BigaQuad features, it says I need to add   

> CFLAGS += -DENABLE_BQ_DECK
in the config.mk file. However, in the latest firmware, I find there is no config.mk, which should be in tools/make/
So I wonder how can I do this setting now?
Thanks in advance!

**Comment by tobbeanton** [comment, marked as answer]
That is outdated documentation... Now we use kbuild. In terminal run "make menuconfig". Then enable "Expansion deck configuration -> Support the BigQuad deck -> Enable Biqquad deck capabilites". Save the config. run "make clean && make -j8"

**Reply by cc299792458** [reply]
I will try it, thanks!
